tests/security/krb5/krb5_crypt_nfs_client.pm

```diff
@@ -0,0 +1,65 @@
+# Copyright © 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test NFS with krb5 authentication and GSS API - client
+# Maintainer: wnereiz <wnereiz@member.fsf.org>
+# Ticket: poo#51560, poo#52388
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use lockapi;
+use mmapi;
+use krb5crypt;    # Import public variables
+
+sub run {
+    select_console 'root-console';
+
+    mutex_wait('CONFIG_READY_NFS_SERVER');
+
+    # Add principal for NFS service and add it to client's keytable
+    assert_script_run "kadmin -p $adm -w $pass_a -q 'addprinc -randkey nfs/$dom_client'";
+    assert_script_run "kadmin -p $adm -w $pass_a -q 'ktadd nfs/$dom_client'";
+
+    systemctl("restart rpc-gssd.service");
+
+    assert_script_run "mkdir -p $nfs_mntdir";
+
```

Comment:
Why so many blank lines in between :)

Reply:
I have no time to adjust the format, and I can not find a standard about how to arrange blank lines. I could remove some later.

Reply:
I removed some lines, but not so many.